### ISSUE TITLE
openrport and neo-rport

### ISSUE THREAD

**Issue by IOTech17** [IOTech17/neo-rport#1]
@KonradKuznicki 

I also started on myside working on keeping rport opensource.  Do you want to merge the effort into one single repository?

We also need to work on the pairing service in case realvnc decide to prevent the existing rport installation from installing/updating the agents. (it is currently deploying the version 1.0.1 of the agents which doesn't work the the latest opensource version as it requires a license).

Thanks.

**Comment by buzzzo** [comment]
There are also other guys trying to keep it alive: https://github.com/openrport/openrport/
This one has also a discord server .

Thx

**Comment by IOTech17** [comment]
@buzzzo Yes I know I trying to ping @KonradKuznicki to see what we do moving forward
